src/interface/src/app/plan/create-scenarios/create-scenarios.component.html

```diff
@@ -10,6 +10,18 @@
           <app-identify-project-areas *ngIf="project_area_upload_enabled" [formGroup]="formGroups[2]">
           </app-identify-project-areas>
           <app-constraints-panel [constraintsForm]="formGroups[1]" [excludedAreasOptions]="excludedAreasOptions"> </app-constraints-panel>
+          <div class="flex-row gap-12">
+            <button
+              mat-raised-button
+              color="primary"
+              [disabled]="!formGroups[0].valid || !formGroups[1].valid || !formGroups[2].valid || generatingScenario"
+              (click)="createScenarioAndProjectAreas()">
+              {{ generatingScenario ? 'GENERATING SCENARIO...' : 'GENERATE' }}
+            </button>
+            <div>
+              Estimated time ?????? 
```

Comment:
Let's leave this now or hide it with an existing flag.  (Don't bother creating another flag.)